friends/src/App.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from "react";
+import axios from "axios";
+import List from "./Components/List";
+import Form from "./Components/Form";
+import { Route } from "react-router-dom";
+import Edit from "./Components/Edit";
+
+import "./App.css";
+
+const url = "http://localhost:5000/friends";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+      name: "",
+      age: null,
+      email: ""
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("http://localhost:5000/friends")
+      .then(response => {
+        console.log(response);
+        this.setState({ friends: response.data });
+      })
+      .catch(err => {
+        console.log(err);
+      });
```

Comment:
Why have an extra step here in your callback? You can just use `console.log` since `console` is an object and `log` a method on it that'll take however many arguments you pass it and log them:
```javascript
.catch(console.log); /* Error object is passed to console.log implicitly, 
just like an Event object being passed implicitly in event handlers for onClicks */
```

Reply:
The same applies for the other callbacks for your `catch`s.